lms/static/scripts/frontend_apps/components/dashboard/DashboardOrganizationFilters.tsx

```diff
@@ -0,0 +1,111 @@
+import { MultiSelect } from '@hypothesis/frontend-shared';
+
+import type { Assignment, Course, Student } from '../../api-types';
+import { useConfig } from '../../config';
+import { useAPIFetch } from '../../utils/api';
+
+export type DashboardFiltersProps = {
+  selectedCourses: Course[];
+  onCoursesChange: (newCourses: Course[]) => void;
+  selectedAssignments: Assignment[];
+  onAssignmentsChange: (newAssignments: Assignment[]) => void;
+  selectedStudents: Student[];
+  onStudentsChange: (newStudents: Student[]) => void;
+};
+
+export default function DashboardOrganizationFilters({
+  selectedCourses,
+  onCoursesChange,
+  selectedAssignments,
+  onAssignmentsChange,
+  selectedStudents,
+  onStudentsChange,
+}: DashboardFiltersProps) {
+  const { dashboard } = useConfig(['dashboard']);
+  const { routes } = dashboard;
+
+  const courses = useAPIFetch<{ courses: Course[] }>(routes.courses);
+  const assignments = useAPIFetch<{ assignments: Assignment[] }>(
+    routes.assignments,
+  );
+  const students = useAPIFetch<{ students: Student[] }>(routes.students);
+
+  return (
+    <div className="flex gap-2 md:w-1/2">
+      <MultiSelect
+        disabled={courses.isLoading}
+        value={selectedCourses}
+        onChange={onCoursesChange}
+        buttonContent={
+          courses.isLoading ? (
+            <>Loading...</>
+          ) : selectedCourses.length === 0 ? (
+            <>All courses</>
+          ) : selectedCourses.length === 1 ? (
+            selectedCourses[0].title
+          ) : (
+            <>{selectedCourses.length} courses</>
+          )
+        }
+      >
+        <MultiSelect.Option value={undefined}>All courses</MultiSelect.Option>
+        {courses.data?.courses.map(course => (
+          <MultiSelect.Option key={`course_${course.id}`} value={course}>
```

Comment:
```suggestion
          <MultiSelect.Option key={course.id} value={course}>
```

The key doesn't have to be globally unique, so `course.id` is fine here.

Reply:
Oh, I think this is a leftover from a previous PR. At some point I got warnings from react due to duplicated keys, I thought it would be a conflict between dropdowns, and it turned out to be duplicated items in the list itself.

Then I forgot to roll back that change 😅